lib/nerves_hub_link/application.ex

```diff
@@ -26,6 +26,8 @@ defmodule NervesHubLink.Application do
 
   defp children(%{connect: false}, _fwup_config), do: []
 
+  defp children(%{firmware_updates: false} = config, _fwup_config), do: [{Socket, config}]
```

Comment:
If we do this, can we do a big fat warning that yells it is disabled with the instruction for turning it back on? (maybe at compile time)

I worry about too many of these developer conveniences making it too easy to prevent a real device from connecting and updating.

Alternatively, if you still want to see the updates attempt, you could also try changing `fwup_devpath: "/tmp/some_bogus_devpath"` - Fwup will write to that temp file and you get all the progress

Reply:
Oh, happy to try that for sure. I believe the explosions I was mitigating were around the fwup_devpath. I think I misread that as fwup_path and just noped out of it.

Reply:
I get an initial thing where the socket crashes on join and then joins clean. I think that's Nerves.Runtime not having the value on the first attempt. But not sure. Good enough to be useful though and we can harden nerves_hub_web to be more graceful potentially.